samples/stockquote/bundle/resources/pom.xml

```diff
@@ -12,7 +12,7 @@
     <carbon.messaging.version>1.0.6</carbon.messaging.version>
     <carbon.transport.http.netty.version>2.1.4</carbon.transport.http.netty.version>
     <carbon.metrics.version>2.0.0</carbon.metrics.version>
-    <msf4j.version>2.1.0-SNAPSHOT</msf4j.version>
+    <msf4j.version>2.1.1-SNAPSHOT</msf4j.version>
```

Comment:
Sample product build fails with 2.1.0-SNAPSHOT version. Cannot find msf4j feature with 2.1.0-SNAPSHOT, since it already released. So I change it to latest version.
Thanks